locales/README.es_ES.md

```diff
@@ -18,13 +18,13 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
 
 > **P:** ¿A qué te refieres con, "Centos ha cambiado de dirección"?
 
-El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualziaciones *antes* de su inclusión en el proveedor upstream).
+El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualizaciones *antes* de su inclusión en el proveedor upstream).
```

Comment:
When it says "recibía parches y actualizaciones luego del proveedor", maybe it's better "recibía parches y actualizaciones después del proveedor"

Reply:
Sorry for the delay! Yes, I agree that "después" is better than "luego".